services/grafana/auth_server.go

```diff
@@ -64,13 +64,15 @@ var rules = map[string]role{
 
 	"/v0/qan/": viewer,
 
+	// mustSetupRules group
 	"/prometheus":      admin,
 	"/victoriametrics": admin,
 	"/alertmanager":    admin,
 	"/graph":           none,
 	"/qan":             none,
 	"/swagger":         none,
 
+	"/logs.zip": admin,
```

Comment:
based on the task description `admin:admin` is already has access to download logs.zip, so we should create a user with Admin level (but not Grafana Admin). I've updated description to explain how to create this type of user.